js/util/dom2.js

```diff
@@ -19,10 +19,11 @@ var el = Element.prototype;
 var doc = Document.prototype;
 var frag = DocumentFragment.prototype;
 var win = Window.prototype;
+var eventTarget = EventTarget.prototype;
```

Comment:
using `EventTarget` prototype instead of window / element, because they both inherit from EventTarget.